src/helpers.js

```
@@ -18,204 +18,204 @@ import update from './modules/update.js'
+
+    return this.length
+  }
+
+  /**
+   * Add multiple values to an array.
+   * Only add the value if the value isnt in the array.
+   * @param {any[]} newValues
+   * @returns {number}
+   */
+  pushMultipleIfNotExists (newValues) {
+    const array = this
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)